biab/start.py

```diff
@@ -0,0 +1,238 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# /usr/bin/env python
+# pylint: disable=E1101
+"""
+Bot-in-a-box: Creates and runs setup for Dialogflow Integration bot
+"""
+
+import subprocess
+import json
+import os
+import re
+from time import sleep
+import firebase_admin
+import requests
+from googleapiclient import discovery
+from google.oauth2.service_account import Credentials
+from firebase_admin import credentials
+from firebase_admin import db
+
+# APIs to enable:
```

Comment:
nit: File should have a description of what it does

Reply:
There is a description at the top of the file "Bot-in-a-box: Creates and runs setup for Dialogflow Integration bot"